packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -11,30 +11,59 @@
  */
 
 import {action} from '@storybook/addon-actions';
-import React, {useState} from 'react';
+import Audio from '@spectrum-icons/workflow/Audio';
+import {Icon} from '@react-spectrum/icon';
+import {Item} from '@react-stately/collections';
+import React from 'react';
 import {storiesOf} from '@storybook/react';
-import {Tag, TagGroup} from '../src';
+import {TagGroup} from '../src';
+import {Text} from '@react-spectrum/text';
 
 storiesOf('TagGroup', module)
   .add(
     'default',
     () => render({})
-  ).add(
+  )
+  .add(
+    'items',
+    () => (
+      <TagGroup aria-label="tag group" items={[{key: '1', label: 'Cool Tag 1'}, {key: '2', label: 'Cool Tag 2'}]}>
+        {item =>
+          <Item key={item.key} aria-label={item.label}><Text>{item.label}</Text></Item>
+        }
+      </TagGroup>
+    )
+  )
+  .add('icons', () => (
+    <TagGroup aria-label="tag group" items={[{key: '1', label: 'Cool Tag 1'}, {key: '2', label: 'Cool Tag 2'}]}>
+      {item => (
+        <Item key={item.key} aria-label={item.label}>
```

Comment:
I don't think we need this `aria-label` on each item. With VoiceOver at least, it leads to double voicing of the item label.